common/src/registers/MachRegister.C

```diff
@@ -947,6 +980,10 @@ namespace Dyninst {
         return false;
       }
 
+      case Arch_riscv64: {
```

Comment:
If Capstone supports them, we should probably include the **un**privileged registers at some point. We don't have to do that now.

Reply:
Unfortunately Capstone does not support them